File: python/7 kyu/31.py
# https://www.codewars.com/kata/5ebd53ea50d0680031190b96/train/python

# Passed

def get_turkish_number(num):
    turkish_number_dict = {
        0:"sıfır",
        1:"bir",
        2:"iki",
        3:"üç",
        4:"dört",
        5:"beş",
        6:"altı",
        7:"yedi",
        8:"sekiz",
        9:"dokuz",
        10:"on",
        20:"yirmi",
        30:"otuz",
        40:"kırk",
        50:"elli",
        60:"altmış",
        70:"yetmiş",
        80:"seksen",
        90:"doksan",
    }
    
    if turkish_number_dict.get(num):
        return turkish_number_dict.get(num)
    
    quotient = num // 10
    remainder = num % 10

    return " ".join([turkish_number_dict.get(quotient * 10), turkish_number_dict.get(remainder)])

output = get_turkish_number(94)
print(output)
        

File: nodejs/7 kyu/31.js
// https://www.codewars.com/kata/55147ff29cd40b43c600058b/train/javascript

// Passed

function charConcat(string) {
  let result = "";
  for (let index = 0; index < Math.floor(string.length / 2); index++) {
    result += `${string[index]}${string[string.length - (index + 1)]}${
      index + 1
    }`;
  }

  return result;
}

const output = charConcat("abcdef");
console.log(output);
